xml2xls/xls2xml.py

```
@@ -132,6 +132,7 @@ def start_convert(options):
 def main():
     options = add_parser()
     start_convert(options)
+    aaaaa()
```

Ruff (0.13.3)

Undefined name `aaaaa`

(F821)